incubating/test-reporting/step.yaml

```diff
@@ -77,21 +76,39 @@ spec:
                 "type": "string",
                 "description": "Directory with test report files"
             },
+            "report_path": {
+                "type": "string",
+                "description": "Path where the report files are saved in the bucket"
+            },
             "report_index_file": {
                 "type": "string",
                 "description": "Root file that will be open in report"
             },
             "branch" : {
                 "type": "string",
                 "description": "Normalized branch name"
+            },
+            "max_upload_size_mb": {
+                "type": "integer",
+                "description": "Max upload size in MB",
+                "default": 1000
+            },
+            "cf_api_retries": {
+                "type": "integer",
+                "description": "The number of times to retry if a Codefresh API call fails",
+                "default": 0
+            },
+            "image_tag": {
+                "type": "string",
+                "description": "The version of test reporting image",
+                "default": "1.2.0"
```

Comment:
self-reminder: make sure the `cf-docker-test-reporting:1.2.0` image is published before merging this PR.